core-bundle/src/Resources/contao/templates/modules/mod_two_factor.html5

```diff
@@ -47,6 +47,36 @@
         </div>
       </form>
     <?php endif; ?>
+    <div class="recovery_codes">
+      <h3><?= $this->backupCodesLabel ?></h3>
```

Comment:
According to our discussion, we should use `$this->trans()` for labels in the template

Reply:
Changed in a22ba09d3ac32e77bc78600b373a50e8bb283dc3.